tests/core/scene/meta-tags.test.js

```diff
@@ -0,0 +1,55 @@
+/* global assert, setup, suite, test */
+var helpers = require('../../helpers');
+var initMetaTags = require('core/scene/metaTags').inject;
+var metaTags = require('core/scene/metaTags');
+
+suite('metaTags', function () {
+  setup(function (done) {
+    var el = this.el = helpers.entityFactory();
+    var i;
+    var metaTagEls = document.querySelectorAll('meta');
+    el.addEventListener('loaded', function () {
+      for (i = 0; i < metaTagEls.length; i++) {
+        metaTagEls[i].parentNode.removeChild(metaTagEls[i]);
+      }
+      done();
+    });
+  });
+
+  test('sets up meta tags', function () {
+    var appleMobileWebAppCapableMetaTag;
+    var sceneEl = this.el.sceneEl;
+    var webAppCapableMetaTag;
+
+    initMetaTags(sceneEl);
+
+    webAppCapableMetaTag = document.querySelector('meta[name="web-app-capable"]');
+    appleMobileWebAppCapableMetaTag = document.querySelector(
+      'meta[name="apple-mobile-web-app-capable"]');
+
+    assert.equal(document.querySelector('meta[name="viewport"]').content,
```

Comment:
to be consistent, we should probably have a `var viewportMetaTag` and assert that exists first